TVOAppliance/hf_upgrade.sh

```diff
@@ -54,12 +54,38 @@ function check_package_status()
         done
 }
 
+#function to restart wlm-cron on primary node only...
+function restart_wlm_cron_on_primary_node()
+{
+	FILE="/etc/tvault-config/tvault-config.conf"
+	#find primary node first.
+	if [ -f "$FILE" ]; then
+		external_virtual_ip=`grep -i 'virtual_ip\ '  /etc/tvault-config/tvault-config.conf | cut -d ' ' -f 3`
+		#check the last command status.
+		if [ $? -eq 0 ]; then
+			current_node_ip=`ip a | grep $external_virtual_ip`
+			#check the last command status.
+			if [ $? -eq 0 ]; then
+				echo Restart wlm-cron service here.
+
+				#This is primary node so disable and enable the wlm-cron service.
+				disable_wlm_cron=`pcs resource disable wlm-cron`
```

Comment:
The commands are executed and output assigned to a variable. This variable is not used anywhere. Will this suppress the command output? If yes, then I believe either we should read those variables ahead (which might be an overhead) OR better let the command run as is so that the output would be printed on console (_or can be redirected to a file_)